### PR TITLE
Create basic app

### PR DESCRIPTION
**Pull request by smesterheide**

### CODE REVIEW

File: .php-cs-fixer.cache [hunk 1]
```diff
@@ -0,0 +1 @@
...:{"tests\/Unit\/Controller\/PageControllerTest.php":2207155758,"tests\/Integration\/AppTest.php":2950225233,"tests\/bootstrap.php":41908297,"appinfo\/routes.php":2458579438,"templates\/adminSettings.php":805129342,"templates\/personalSettings.php":1395204514,"templates\/index.php":2734865279,"node_modules\/flatted\/php\/flatted.php":3670498728,"lib\/Settings\/Admin.php":4177689181,"lib\/Settings\/AdminSection.php":3667266050,"lib\/Settings\/Personal.php":774712035,"lib\/Settings\/PersonalSection.php":237427933,"lib\/Vendor\/Firebase\/JWT\/BeforeValidException.php":3387887423,"lib\/Vendor\/Firebase\/JWT\/Key.php":2086644246,"lib\/Vendor\/Firebase\/JWT\/ExpiredException.php":1091148461,"lib\/Vendor\/Firebase\/JWT\/SignatureInvalidException.php":3046842981,"lib\/Vendor\/Firebase\/JWT\/JWT.php":23558362,"lib\/Vendor\/Firebase\/JWT\/JWK.php":909127881,"lib\/Controller\/LoginController.php":2413542052,"lib\/Controller\/PageController.php":4113731223,"lib\/Controller\/ConfigController.php":2824957717,"lib\/Service\/ProviderService.php":661513495,"lib\/AppInfo\/Application.php":3846310732,"lib\/Backend\/GroupBackend.php":986210557,"lib\/Service\/VirtualOrganisationService.php":3422386287}}
```
Comment (come-nc) [inline .php-cs-fixer.cache:1]: This file should not be commited to git.
Same thing for the vendor-bin folder.
Maybe add a .gitignore file for this?

File: lib/Backend/GroupBackend.php [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+ * Abstract base class for user management
+ */
+class GroupBackend extends ABackend implements
+	IGetDisplayNameBackend,
+	IGroupDetailsBackend {
+
+	/**
+	 * @var string The application name.
+	 */
+	private $appName;
```
Comment (come-nc) [inline lib/Backend/GroupBackend.php:43]:
```suggestion
	private string $appName;
```

Please strong type properties

File: lib/Backend/GroupBackend.php [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+	/**
+	 * @var ILogger The logger instance.
+	 */
+	private $logger;
```
Comment (come-nc) [inline lib/Backend/GroupBackend.php:47]:
```suggestion
	private LoggerInterface $logger;
```
ILogger is deprecated, use `Psr\Log\LoggerInterface` instead

**Comment by smesterheide** [comment]
Closed in favor of https://github.com/nextcloud/vo_federation/pull/16

**Pull request closed without merging**